src/node_platform.cc

```diff
@@ -27,12 +24,51 @@ static void BackgroundRunner(void* data) {
   }
 }
 
-NodePlatform::NodePlatform(int thread_pool_size, uv_loop_t* loop,
-                           TracingController* tracing_controller)
-    : loop_(loop) {
-  CHECK_EQ(0, uv_async_init(loop, &flush_tasks_, FlushTasks));
-  flush_tasks_.data = static_cast<void*>(this);
-  uv_unref(reinterpret_cast<uv_handle_t*>(&flush_tasks_));
+PerIsolatePlatformData::PerIsolatePlatformData(
+    v8::Isolate* isolate, uv_loop_t* loop)
```

Comment:
Nit: Perhaps use an unqualified name for the `Isolate` type here?

Reply:
Since this is a backport is it safe to assume this is what already landed upstream

Reply:
Yeah, I’d prefer to avoid stylistic changes outside of `master` :)